pkg/operator/resourceapply_missing.go

```diff
@@ -0,0 +1,39 @@
+package operator
+
+import (
+	"context"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resource/resourcehelper"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	admissionregistrationclientv1 "k8s.io/client-go/kubernetes/typed/admissionregistration/v1"
+)
+
+// TODO: delete once available in github.com/openshift/library-go
+// see: https://github.com/openshift/library-go/pull/1954
+
+func DeleteValidatingAdmissionPolicyV1(ctx context.Context, client admissionregistrationclientv1.ValidatingAdmissionPoliciesGetter, recorder events.Recorder, required *admissionregistrationv1.ValidatingAdmissionPolicy) (*admissionregistrationv1.ValidatingAdmissionPolicy, bool, error) {
+	err := client.ValidatingAdmissionPolicies().Delete(ctx, required.Name, metav1.DeleteOptions{})
+	if err != nil && apierrors.IsNotFound(err) {
+		return nil, false, nil
+	}
+	if err != nil {
+		return nil, false, err
+	}
+	resourcehelper.ReportDeleteEvent(recorder, required, err)
+	return nil, true, nil
+}
```

Comment:
This (and the next function) always have `nil` as the first returned value.  Is that to keep it compatible with what we currently do in `library-go` ?

Reply:
Exactly, check for instance: https://github.com/openshift/library-go/blob/ce2ba53fb2a4b7318769480e897a0e506b7cb0c0/pkg/operator/resource/resourceapply/rbac.go#L206-L252 or https://github.com/openshift/library-go/blob/ce2ba53fb2a4b7318769480e897a0e506b7cb0c0/pkg/operator/resource/resourceapply/core.go#L628-L698 
Honestly I'm also not a big fan of that but I understand that keeping the same signatures between create and delete functions is a nice to have.

Reply:
OK, all good.